privacy_consent/data/mail.xml

```diff
@@ -42,28 +42,33 @@
                                 style="padding:15px 20px 0px 20px; font-size:16px;"
                             >
                                 <p>
-                                    Hello, ${object.partner_id.name|safe}
+                                    Hello, <t t-out="object.partner_id.name" />
```

Comment:
Why not `Hello, {{object.partner_id.name}}`?

Reply:
And the same applies to the rest.

Reply:
According to v15 guidelines and all odoo examples they cannot be used in mail body.